src/megaclient.cpp

```diff
@@ -4814,6 +4818,10 @@ void MegaClient::sc_chatupdate()
                     chat->url = ""; // not received in action packets
                     chat->ou = ou;
                     chat->title = title;
+                    if (ts != -1)
```

Comment:
I have a doubt about this.
In this case, `chat->ts` is not updated if it's not received. However, in other two places in this pull request, `chat->ts` is set to 0 when it's not received.
I think that it's OK, but just to be sure that it's the intended.